superbench/benchmarks/micro_benchmarks/micro_base.py

```diff
@@ -39,14 +42,40 @@ def _preprocess(self):
         """
         return super()._preprocess()
 
-    @abstractmethod
     def _benchmark(self):
         """Implementation for benchmarking.
 
         Return:
             True if run benchmark successfully.
         """
-        pass
+        for command in self._commands:
```

Comment:
How can I process the results? The results from all _commands will be merged into one?

Reply:
The raw result parsing is depend on benchmark owner. For example, P2P bandwidth, Kernel Launch Overhead, benchmark owner should parse the raw string in _process_raw_result(). The results from all commands will be parsed and regarded as different metrics. Most benchmarks just have one command.

Reply:
Got it. To my understanding, if we have multiple commands, they will merge into one for the ooutput.stdout. Correct? Therefore, it should do a well design for the _process_raw_result().